src/AddTodoForm.js

```diff
@@ -1,12 +1,40 @@
-import React from "react";
+import React, { useState } from "react";
+
+function AddTodoForm({ onAddTodo }) {
+    const [todoTitle, setTodoTitle] = useState(''); 
+
+    function handleAddTodo(event) {
+        event.preventDefault();
+        const newTodo = {
+            title: todoTitle,
+            id: Date.now()
+        };
+        onAddTodo(newTodo);
```

Comment:
No need to change it if the explicit steps make it easier for you to follow along with what's happening, but this could also be condensed to this
```suggestion
        onAddTodo({
            title: todoTitle,
            id: Date.now()
        });
```